modules/apps/remote-app/remote-app-support-web/test/index.js

```diff
@@ -163,4 +163,56 @@ describe('remote-app-support-web', () => {
 			expect(receiveMessage).not.toHaveBeenCalled();
 		});
 	});
+
+	describe('"fetch" command', () => {
+		it('doesn\'t throw an error if "body" is undefined', async () => {
+			const headers = new Headers();
```

Comment:
Brian might ask for a blank line here. See: https://github.com/liferay/liferay-frontend-projects/issues/15